src/libutil/args.hh

```diff
@@ -155,6 +155,8 @@ protected:
      */
     using CompleterClosure = std::function<CompleterFun>;
 
+public:
+
```

Comment:
`Flag` type needs to be public for defs in `libcmd`. I think that is fine since `addFlag` is already public --- making the parameter type of a `public` function `protected` seems like a mistake!